docs/release_notes/pending_release.rst

```diff
@@ -19,3 +19,7 @@ Web
 
 Fixes
 -----
+
+CI
+
+ * Fix authentication issue during docker push action step
```

Comment:
This sort of PR doesn't need a change note, because we're "fixing" something that we note as a new in the above section.

Reply:
Yeah, I thought it looked kind of weird. Just wasn't sure how to handle the ghostflow error about the missing release_notes or if we could just ignore it.